WordPress/Classes/Models/Media.h

```diff
@@ -70,6 +70,8 @@ typedef NS_ENUM(NSUInteger, MediaType) {
 /**
  Local file URL for a preprocessed **large** thumbnail that can be used for
  a full-screen presentation.
+
+ - warning: Deprecated! Use ``MediaImageService`` to access the thumbnails.
  */
 @property (nonatomic, strong, nullable) NSURL *absoluteThumbnailLocalURL;
```

Comment:
It's used only by the editor and in Post Settings for uploading featured images. I tested the latter and it also works, but it should not be using this property, and I'll open a separate PR to update it.